app/bundles/EmailBundle/Form/Type/ScheduleSendType.php

```diff
@@ -0,0 +1,132 @@
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+/**
+ * @extends AbstractType<ScheduleSendType>
+ */
+final class ScheduleSendType extends AbstractType
+{
+    public function __construct(private TranslatorInterface $translator)
+    {
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder->add(
+            'publishUp',
+            DateTimeType::class,
+            [
+                'widget' => 'single_text',
+                'label'  => 'mautic.email.send.datetime.start',
+                'attr'   => [
+                    'class'       => 'form-control',
+                    'data-toggle' => 'datetime',
+                ],
+                'format'      => 'yyyy-MM-dd HH:mm',
+                'required'    => false,
+                'html5'       => false,
+                'constraints' => new NotBlank(
+                    [
+                        'message' => 'mautic.core.value.required',
+                    ]
+                ),
+            ]
+        );
+
+        $builder->add(
+            'continueSending',
+            YesNoButtonGroupType::class,
+            [
+                'label'    => 'mautic.email.send.continue',
+                'required' => false,
+                'attr'     => [
+                    'tooltip' => 'mautic.email.send.continue.tooltip',
+                ],
+                'data'     => $options['data']['continueSending'] ?? false,
+            ]
+        );
+
+        $builder->add(
+            'publishDown',
+            DateTimeType::class,
+            [
+                'widget' => 'single_text',
+                'label'  => 'mautic.email.send.datetime.end',
+                'attr'   => [
+                    'class'            => 'form-control',
+                    'data-toggle'      => 'datetime',
+                    'data-show-on'     => '{"schedule_send_continueSending_1":"checked"}',
+                ],
+                'format'      => 'yyyy-MM-dd HH:mm',
+                'html5'       => false,
+                'required'    => false,
+                'constraints' => [
+                    new Callback(function ($value, ExecutionContextInterface $context): void {
+                        $data            = $context->getRoot()->getData() ?? [];
+                        $continueSending = $data['continueSending'] ?? false;
+                        $publishUp       = $data['publishUp'] ?? null;
+                        if ($continueSending && $value && $publishUp && $value <= $publishUp) {
+                            $context->buildViolation($this->translator->trans('mautic.form.date_time_range.invalid_range', [], 'validators'))
+                                ->addViolation();
+                        }
+                    }),
+                ],
```

Comment:
Please redefine the constraint from a form type to the entity itself so it would be validated also via API.